wiki/en/en-Software-Manual.md

```diff
@@ -197,7 +193,7 @@ therefore influences the quality of the audio stream (how many dropouts occur) a
 
 You can set the jitter buffer size manually for your local client and the remote server. For the local jitter
 buffer, dropouts in the audio stream are indicated by the light below the jitter buffer size faders.
-If the light turns to red, a buffer overrun/underrun has taken place and the audio stream is interrupted.
+If the light turns to red, a buffer over-/under- run has taken place and the audio stream is interrupted.
```

Comment:
Overrun/underrun are correct. Maybe it could be simplified with 'over/underrun'.

Reply:
`overrun/underrun` is much more comprehensible for non-native readers IMHO.